tests/api/test_instance.py

```diff
@@ -90,24 +85,12 @@ def test_chronos_instance_status(
         'fake_cluster2.fake_instance2': 'GIT_SHA',
     }
     mock_validate_service_instance.return_value = 'chronos'
-    mock_select_tasks_by_id.return_value = [1, 2, 3]
-
-    mock_load_chronos_job_config.return_value = ChronosJobConfig(
-        'fake_service',
-        'fake_instance',
-        'fake_cluster',
-        {
-            'schedule': 'always',
-        },
-        None,
-    )
 
     request = testing.DummyRequest()
     request.swagger_data = {'service': 'fake_service', 'instance': 'fake_instance'}
 
-    response = instance.instance_status(request)
-    assert response['chronos']['schedule']['schedule'] == 'always'
-    assert response['chronos']['schedule_type'] == 'schedule'
+    instance.instance_status(request)
+    assert mock_status_chronos_jobs.called
```

Comment:
You could assert `mock_status_chronos_jobs.call_args` are what you expect, to check the cluster, etc.